go.mod

```diff
@@ -19,9 +19,9 @@ require (
 	gorm.io/datatypes v1.0.5
 	gorm.io/driver/mysql v1.3.5 // indirect
 	gorm.io/driver/postgres v1.2.3 // indirect
-	gorm.io/driver/sqlite v1.3.6
+	gorm.io/driver/sqlite v1.4.4
 	gorm.io/driver/sqlserver v1.2.1 // indirect
-	gorm.io/gorm v1.23.8
+	gorm.io/gorm v1.24.5
```

Comment:
The GORM update allegedly contains a bugfix for cascading delete behavior for many2many relationships.